regulation-worker/internal/client/client.go

```diff
@@ -37,12 +37,19 @@ func (j *JobAPI) Get(ctx context.Context) (model.Job, error) {
 		return model.Job{}, err
 	}
 
+	workspaceToken := config.GetEnv("CONFIG_BACKEND_TOKEN", "22JnFdS3ZKDd0UfuEYowUeNi9fe")
```

Comment:
lets not hardcode the config.GetEnv here.

Reply:
removed the `default` with `""`. And, panic if `workspaceToken` is empty.